basics.js

```diff
@@ -31,6 +40,18 @@ def greet(name, excited)
 end
 */
 
+var greet = function(name, excited){
+  var greeting = ("Hi, " + name + " here!");
+  if (excited === true) {
+    console.log(greeting.toUpperCase());
```

Comment:
This is doing the same as a `puts` in Ruby, and is returning a value of `undefined` instead of the string itself.